Raspi/Test/mock_wireless_module.py

```diff
@@ -166,10 +170,44 @@ def publish(client, topic, data):
     print(topic, "--> ", json_data)
 
 
+def start_modules(args):
+    """ Send the a fake filename on the start channel to start the appropriate
+    module"""
+
+    if args.id is None:
+        for i in range(1, 4):
```

Comment:
This should be made to take in N number of modules. Not just 3. Perhaps create another argument where we can pass in a number to create N number of modules. This will ignore/overwrite the id argument.

Reply:
If N number of modules are selected should this just make up what sensors go on each module. Because at the current moment the sensors are hard coded for each module to represent the actual sensors.

Reply:
Good question. Ignore me for now. But we should create another card that allows the user to select number of modules + what module to include. Perhaps the user should be allowed to input a config file that define module ID and sensors on that module?

Reply:
Create a TODO on the for loops that define the above behaviour I described.